src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +45,17 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            return session.createQuery("FROM Movie", Movie.class)
+                    .getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get all movies", e);
```

Comment:
Consider using `DataProcessingException` instead of `RuntimeException` for consistency with other methods. This will ensure uniform exception handling across the DAO implementation.